model/src/main/resources/reference.yml

```diff
@@ -202,44 +202,46 @@ ort:
           sslrootcert: /defaultdir/root.crt
           parallelTransactions: 5
 
-    options:
-      # A map of maps from scanner class names to scanner-specific key-value pairs.
+    config:
+      # A map from scanner plugin types to the plugin configuration.
       ScanCode:
-        # Command line options that affect the ScanCode output. If changed, stored scan results that were created with
-        # different options are not reused.
-        commandLine: '--copyright --license --info --strip-root --timeout 300'
+        options:
+          # Command line options that affect the ScanCode output. If changed, stored scan results that were created with
+          # different options are not reused.
+          commandLine: '--copyright --license --info --strip-root --timeout 300'
 
-        # Command line options that do not affect the ScanCode output.
-        commandLineNonConfig: '--processes 4'
+          # Command line options that do not affect the ScanCode output.
+          commandLineNonConfig: '--processes 4'
 
-        # Criteria for matching stored scan results. These can be configured for any scanner that uses semantic
-        # versioning. Note that the 'maxVersion' is exclusive and not part of the range of accepted versions.
-        minVersion: '3.2.1-rc2'
-        maxVersion: '32.0.0'
+          # Criteria for matching stored scan results. These can be configured for any scanner that uses semantic
+          # versioning. Note that the 'maxVersion' is exclusive and not part of the range of accepted versions.
+          minVersion: '3.2.1-rc2'
+          maxVersion: '32.0.0'
 
       FossId:
-        serverUrl: 'https://fossid.example.com/instance/'
-        user: user
-        apiKey: XYZ
+        options:
+          serverUrl: 'https://fossid.example.com/instance/'
+          user: user
+          apiKey: XYZ
```

Comment:
These should be secrets, I believe.

Reply:
Ok, I see this happens in the next commit. I was expecting it to already happen in this commit as the commit message talk about secrets options.

Reply:
Yes, this commit is only about changing the structure of the scanner configuration to support secret options.